### DISCUSSION TITLE
Possibility to install it manually

### DISCUSSION THREAD

**Discussion by BMWfan** [homeassistant-apps/app-cloudflared#282] [Q&A] [answered]
Hey guys,

im using homeassistant in a docker container thatswhy i have no possibility to install this add-on manually by adding it over my HA-Addons repository.

Is there another possibility to install it manually and get it running over docker compose or something?

Greetings

Daniel

**Comment by elcajon** [comment, marked as answer]
Hey @BMWfan, 
it's not possible to run this add-on without a supervised or operating system installation. But you can simply install cloudflared and spin up the tunnel. Just follow this instructions: https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/
